provision/minikube/keycloak/templates/keycloak.yaml

```diff
@@ -0,0 +1,97 @@
+apiVersion: k8s.keycloak.org/v2alpha1
+kind: Keycloak
+metadata:
+  labels:
+    app: keycloak
+  name: keycloak
+  namespace: {{ .Values.namespace }}
+spec:
+  hostname: keycloak.{{ .Values.hostname }}
+  serverConfiguration:
+    - name: db
+      value: postgres
+    - name: db-url
+      value: jdbc:postgresql://postgres:5432/keycloak
+    - name: log-console-output
+      value: json
+    - name: metrics-enabled
+      value: 'true'
+    - name: health-enabled
+      value: 'true'
+    - name: db-username
+      secret:
+        name: keycloak-db-secret
+        key: username
+    - name: db-password
+      secret:
+        name: keycloak-db-secret
+        key: password
+  tlsSecret: keycloak-tls-secret
+  unsupported:
+    podTemplate:
+      spec:
+        containers:
+          - name: keycloak
```

Comment:
It just feels strange to had a container without a name ... 

What if I would add a second (new) container with a different name, ie a sidecar? How would that / should that behave?

Reply:
Will remove the name in the next commit.

Reply:
> What if I would add a second (new) container with a different name, ie a sidecar? How would that / should that behave?

It will be appended to the `podTemplate` and work as expected (the operator won't mangle anything).
The idea behind having a few fields "protected" is that the operator "owns" the container 0, do you have any suggestion/feedback in this regard?